### ISSUE TITLE
On Google Chrome keyboard navigation is not  visible 

### ISSUE THREAD

**Issue by sbadau** [mozilla/blurts-server#1603]
**Affected versions**
- Google Chrome Version 80.0.3987.149 
                                                                                                                                                                                                                                             
**Affected platforms**
- Windows 10 and 7 x64
- Mac OS X 10.14
- Ubuntu 18.04 

**Steps to reproduce**
1. Go to https://blurts-server-review.herokuapp.com
2. In the upper side of the page, click on Breaches
3. Click on a random breach
4. Press the "Tab" key to navigate to the "Monitor Sign Up Promo" banner.

**Expected behavior**
Keyboard navigation is allowed and a focus ring should show the location on the page.

**Actual behavior**
- Keyboard navigation is allowed and the Ecosystems Banners are reached, but it's not visible at all since there is no focus ring to indicate it.
Please see the video for more details and note that the only way I am navigating on the page is by pressing the Tab key.
It is also noticeable in the screencast, that the Links tooltips are changed.

![2020-03-19_12h06_19](https://user-images.githubusercontent.com/56026823/77055879-4b940400-69da-11ea-8d72-ad66cffc71b5.gif)

**Comment by EMMLynch** [comment]
Closing since we've redesigned the site and functionality since this was created. If you feel that this is still needed, please let me know.
